src/WordPress/Blueprints/functions.php

```diff
@@ -42,16 +44,18 @@ function list_files( string $path, $omitDotFiles = false ): array {
 }
 
 function move_files_from_directory_to_directory( string $from, string $to ) {
+	$fs = new Filesystem();
 	$files = scandir( $from );
 	foreach ( $files as $file ) {
 		if ( '.' === $file || '..' === $file ) {
 			continue;
 		}
-		$fromPath = $from . '/' . $file;
-		$toPath = $to . '/' . $file;
-		$success = rename( $fromPath, $toPath );
-		if ( ! $success ) {
-			throw new IOException( "Failed to move the file from {$fromPath} at {$toPath}" );
+		$fromPath = Path::canonicalize($from . '/' . $file);
```

Comment:
I'm not sure this is the right approach as it will turn `C:\Program Files\Stuff` into `C:/Program Files/Stuff`. It might be fine but I'm not sure if it will universally work with all Windows versions and paths, e.g. what about network shared directories such as `\\NetDirectory\SubPath`? 

That being said, perhaps it's good enough for now. Would you please start a new issue to get to the bottom of this eventually?